static/templates/navbar_alerts/navbar_alert_wrapper.hbs

```diff
@@ -0,0 +1,11 @@
+<div data-process="{{data_process}}" class="alert alert-info {{custom_class}}">
+    <span class="close" data-dismiss="alert" aria-label="{{t 'Close' }}" role="button" tabindex=0>&times;</span>
+</div>
+
+{{#if is_bankruptcy_alert}}
+<div class="alert alert-info bankruptcy-loader" style="display: none;">
+    {{#tr}}
+    Marking all messages as read…
+    {{/tr}}
+</div>
+{{/if}}
```

Comment:
@Riken-Shah could we move this to be some sort of step-2 element in `bankruptcy.hbs` or something as a follow-up?

It doesn't really below in the top-level template (and actually I think there's no reason it couldn't be inside the main `alert` property.

Reply:
We can definitely move it as a `step-2` element.

I'll have opened the [PR](https://github.com/zulip/zulip/pull/18739) for it!